pkg/datapath/loader/loader.go

```diff
@@ -713,6 +713,9 @@ func endpointRewrites(ep datapath.EndpointConfiguration, lnc *datapath.LocalNode
 	cfg.EnableICMPRule = option.Config.EnableICMPRules
 	cfg.EnableLRP = option.Config.EnableLocalRedirectPolicy
 
+	// To support per-endpoint conntrack accounting, override it in node_config
+	cfg.EnableConntrackAccounting = ep.GetOptions().IsEnabled(option.ConntrackAccounting)
+
```

Comment:
https://github.com/cilium/cilium/pull/43147/files#diff-ee98a7dc5cb83d0ccacc236a517ed1c3e605c84d897de0ced03e331ccf451edf

According to the documentation, the scope has changed from `node or per-endpoint` to `node`.
Should we continue to maintain support for the previously supported per-endpoint option, or should it be removed?